e2eTests/chrome/hideElements/hideElements.test.js

```diff
@@ -0,0 +1,43 @@
+/* globals expect */
+
+import { execSync } from 'child_process';
+import path from 'path';
+import fs from 'fs';
+import config from './hideElementsConfig';
+
+function cleanState(dir) {
```

Comment:
i think you are meant to call this from the before each 😂 
You deleted the comment instead of uncommenting

Reply:
yeah :-D